### DISCUSSION TITLE
[6.10.1] Blocked by MacAfee malware detected

### DISCUSSION THREAD

**Discussion by UnleashSpirit** [cyrilletuzi/vscode-angular-schematics#928] [Q&A] [answered]
Hi,

I try to update to 6.10.1 but the MacAfee Web Gateway of my company is blocking the download cause of virus detection.

Virus Name: JS/Downloader.eu
File Name: cyrilletuzi.angular-schematics-6.10.1.vsix

I got no problem to download 6.10.0 verison

**Comment by cyrilletuzi** [comment]
Hello @UnleashSpirit,

Thank you for the report.

Given how the extension is built, it sometimes happens that it results into false positives in some anti-virus softwares.

Problem is: it happens randomly and it is not an issue on my machines (Mac and a Windows machine where Microsoft Defender is OK). Until now, VS Code marketplace, which does its own anti-virus detection, was flagging the problematic builds, but it was OK this time.

I released v6.11, can you tell me if it is OK?

**Comment by UnleashSpirit** [comment]
Hi, same with 6.11.0

**Comment by cyrilletuzi** [comment]
@UnleashSpirit do you know if there is an online MacAfee service to scan a file?

**Reply by UnleashSpirit** [reply]
Absolutely no idea :(

**Comment by cyrilletuzi** [comment]
OK, I will look into a solution, until then v6.10.0 is totally fine, v6.10.1 was just an internal fix with no user facing changes.

**Comment by cyrilletuzi** [comment, marked as answer]
Closing because I opened a real issue: https://github.com/cyrilletuzi/vscode-angular-schematics/issues/929
